patch/series

```diff
@@ -71,6 +71,8 @@ Support-for-fullcone-nat.patch
 #
 
 0001-efi-memattr-Ignore-table-if-the-size-is-clearly-bogus.patch
+# Backport from v6.14 for v6.1 and v6.12
```

Comment:
Thank you for adding this comment, but the patch is actually in released versions, but the SONiC one is older then that. I’d prefer to have in the comment, the releases the patch was added to the series.

Reply:
Do you mean specifying that the patch is in 6.1.132 and 6.12.20? (basically specifying the specific patch version)

Web link to make it a bit easier to find out which tags a commit has been backported to: https://kernel.dance/#e92df790d07a8eea873efcb84776e7b71f81c7d5

Reply:
Thank you @saiarcot895
I added the comment to highlight the major kernel versions of the most recent released and upcoming sonic images based on debian 12 and 13.